packages/section/stories/it-section.stories.ts

```diff
@@ -0,0 +1,245 @@
+import type { StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { SECTION_VARIANTS, type SectionProps } from '../src/types.ts';
+
+import '@italia/section';
+
+const defaultContent = html`
+  <!-- contenuto di esempio START -->
+  <div class="container">
+    <div class="row mb-3">
+      <div class="col-12">
+        <h2>Morbi fermentum amet</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 col-lg-6 col-xl-4">
+        <p>
+          Platea dictumst vestibulum rhoncus est pellentesque elit ullamcorper dignissim cras. Dictum sit amet justo
+          donec enim diam vulputate ut. Eu nisl nunc mi ipsum faucibus.
+        </p>
+      </div>
+      <div class="col-12 col-lg-6 col-xl-4">
+        <p>
+          Eget egestas purus viverra accumsan. Diam maecenas ultricies mi eget mauris pharetra et. Etiam dignissim diam
+          quis enim. Eu nisl nunc mi ipsum faucibus.
+        </p>
+      </div>
+      <div class="col-12 col-lg-6 col-xl-4">
+        <p>
+          Euismod lacinia at quis risus sed vulputate. Scelerisque purus semper eget duis at tellus at urna condimentum.
+          Mattis enim ut tellus elementum sagittis.
+        </p>
+      </div>
+    </div>
+  </div>
+  <!-- contenuto di esempio END -->
+`;
+function renderSection({
+  variant,
+  image,
+  content,
+  inverse = false,
+}: SectionProps & {
+  content?: any;
+}) {
+  return html`
+    <it-section variant="${variant || nothing}" image="${image || nothing}" ?inverse="${ifDefined(inverse) || nothing}">
```

Comment:
nella condizione dell'attributo inverse, non è sufficiente la ifDefined?

Reply:
ottima domanda, sono andata dritta in autocomplete, verifico dalla doc